readme.md

```diff
@@ -71,7 +71,11 @@ There are two parts to creating an Application Link. The first is done in Jenkin
 To register a consumer: 
 1. In Jenkins go to **Jenkins** > **Manage Jenkins** > **Manage Bitbucket Server consumers**. 
 2. Select **Register new consumer**. 
-3. Enter consumer details. 
+3. Enter these details: 
+   - Consumer key: A unique identifier for your Bitbucket instance. 
+   - Consumer name: A human-readable identifier for your Bitbucket instance.  
+   - Consumer secret: A password Jenkins can use with your key to identify your Bitbucket instance. Once this is entered you won't be able to retrieve it. 
```

Comment:
Same here. I think that maybe this should be something like "Create a password" to indicate its something they have to create